transformation/transformationPlugins/jsonMessageTransformers/jsonTypeMappingsSanitizationTransformer/build.gradle

```diff
@@ -5,13 +5,15 @@ plugins {
 }
 
 dependencies {
-    api project(':transformation:transformationPlugins:jsonMessageTransformers:jsonLiquidJSTransformer')
+    api project(':transformation:transformationPlugins:jsonMessageTransformers:jsonJSTransformer')
 
     implementation project(':transformation:transformationPlugins:jsonMessageTransformers:jsonMessageTransformerInterface')
     implementation group: 'com.fasterxml.jackson.core', name: 'jackson-annotations'
     implementation group: 'com.fasterxml.jackson.core', name: 'jackson-core'
     implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind'
     implementation group: 'com.google.guava', name: 'guava'
+    implementation group: 'org.slf4j', name: 'slf4j-api'
+    implementation group: 'org.apache.logging.log4j', name: 'log4j-slf4j2-impl'
```

Comment:
This needs to be removed - we dont want any log4j dependencies on libraries.  The logging APIs should go through slf4j and additional configuration should be via callbacks or invocations on the providers to orchestrate this (and those shouldn't have a dependency either).